### PR TITLE
Also test with pekko 1.1

### PR DESCRIPTION
**Pull request by pjfanning**
* need to test with pekko 1.0 and 1.1

### CODE REVIEW

File: .github/workflows/nightly-pekko-1.1-builds.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 8
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.4.0
+
+      - name: "compile, including  tests. Run locally with: sbt +Test/compile"
+        run: sbt +Test/compile
```
Comment (He-Pin) [inline .github/workflows/nightly-pekko-1.1-builds.yaml:39]: add a new line here

**Review by He-Pin — APPROVED** [review]
lgtm

File: .github/workflows/nightly-pekko-1.1-builds.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  # Only run once for latest commit per ref and cancel other (previous) runs.
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  style-compile-mima:
+    name: Compile
+    runs-on: ubuntu-20.04
+    env:
+      JAVA_OPTS: -Xms2G -Xmx2G -Xss2M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
```
Comment (He-Pin) [inline .github/workflows/nightly-pekko-1.1-builds.yaml:20]: Where the version 1.1.x is setted?

Comment (pjfanning) [reply .github/workflows/nightly-pekko-1.1-builds.yaml:20]: thanks - added

Comment (pjfanning) [reply .github/workflows/nightly-pekko-1.1-builds.yaml:20]: added to the sbt command below

File: .github/workflows/nightly-pekko-1.1-builds.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 8
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.4.0
+
+      - name: "compile, including  tests. Run locally with: sbt +Test/compile"
+        run: sbt -Dpekko.build.pekko.version=1.0.x +Test/compile
```
Comment (He-Pin) [inline .github/workflows/nightly-pekko-1.1-builds.yaml:39]: I see, so you plan to have a 1.1.0-M1 release then :)

**Review by He-Pin — APPROVED** [review]
lgtm

**Comment by He-Pin** [comment]
@pjfanning Do we need a runner for Java 21 too?

File: .github/workflows/nightly-pekko-1.1-builds.yaml (hunk 3 above)
Comment (pjfanning) [reply .github/workflows/nightly-pekko-1.1-builds.yaml:39]: part of preparing for pekko 1.1 is to start testing it

**Comment by pjfanning** [comment]
> @pjfanning Do we need a runner for Java 21 too? People may jump from Java 8 to Java 21, I think test these two should be OK.

I've updated the 1.1 job to test with multiple JDKs

**Comment by mdedetrich** [comment]
@pjfanning Is it possible to do the source code changes (i.e. moving the `deprecated`) in another PR rather than this one? If its not related to adding testing for Pekko 1.0/1.1 I would rather keep the commits separate otherwise its confusing when we need to look back.

**Comment by pjfanning** [comment]
@mdedetrich I've dropped the code changes for java 17/21. They will be done later.

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**